modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -48,6 +48,10 @@ Specify each range in CIDR notation or as starting and ending IP addresses separ
 Specify `false` if you want explicitly request an IP address from this pool with the `metallb.universe.tf/address-pool` annotation.
 The default value is `true`.
 
+|`avoidBuggyIPs`
```

Comment:
It should say `spec.avoidBuggyIPs` in order to be consistent with the rest of the fields.